text/0000-project-groups.md

```diff
@@ -0,0 +1,328 @@
+  present.
+- Go to step 2.
+
+[`rust-lang/team`]: https://github.com/rust-lang/team
+[internals]: https://internals.rust-lang.org
+
+# Reference-level explanation
+
+A Project Group is a group of people working on a particular project or
+responsibilities at the behest of an official Rust team. Some project groups are
+are ephemeral, meaning that they are archived once the project is complete.
+However, there are project groups that have continual work and maintenance.
+
+Examples of project groups around specific feature  include [FFI Unwind],
+[Inline ASM], and [Safe Transmute]. Examples built around continual work and
+maintenance would include [Triage], and [Rustup].
+
+The goal of a project is build a community or formalise and existing community
+around a particular feature or project in the in the organisation, and use this
+space to discuss and iterate on that feature.
+
+Part of building a community is removing some of the institutional memory that
+develops in the design process, and centralising the information and discussion
+around the feature so that we can provide better visibility into why certain
+decisions and trade offs were made over others.
+
+Previously a lot of the discussion and iteration for large features would
+happen in the initial RFC thread. This leads to a lot of discussion in the top
+of the thread and that often becomes completely irrelevant to the current
+iteration.
+
+This process has also been unsuitable to describe features that can take
+multiple years to develop and will become multiple RFCs over the course of its
+design process. Some examples of of this are the "`impl Trait`" and "macros
+2.0" features, where the goals has shifted a lot from the initial RFCs, and it
+can be hard to know their current status.
+
+[ffi unwind]: https://github.com/rust-lang/project-ffi-unwind
+[inline asm]: https://github.com/rust-lang/project-inline-asm
+[safe transmute]: https://github.com/rust-lang/project-safe-transmute
+[triage]: https://github.com/rust-lang/team/blob/master/teams/wg-triage.toml
+[rustup]: https://github.com/rust-lang/team/blob/master/teams/wg-rustup.toml
+
+## Creation
+
+Since project groups are approved by their relevant parent team over the core
+team, it's up to each team decide their specific requirements. However the author
+recommends that a group should try to address at least of the following questions.
+
+- What value do you see your group bringing to the organisation?
+- What are the goals of your group?
+  - Both in the short term, and if relevant over a longer period.
+- What are explicitly non-goals of your group?
+
+These first three points should also serve as the basis for the charter.
+
+- Is your group long-running or temporary?
+- If it is temporary, how long do you see it running for?
+  - What is the long-term vision of your group?
+- What do you expect the relationship to the organisation be?
+- How do you want to establish accountability?
```

Comment:
I am uncertain exactly what we are seeking for the answer to this question. Could you provide an example?

Reply:
This question was more important for a working group which is more independent. For project groups I don't know if it's relevant as it can (and would probably want to) defer to the parent team.